pom.xml

```diff
@@ -105,7 +105,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
-            <version>1.22</version>
+            <version>2.1.8</version>
```

Comment:
Yes. if you want to go down as far as 2.1.6 you can... but the diff between 2.1.6 and 2.1.8 is recommended given that I have not seen any issues in 2.1.8 in over 2 weeks